satellite/orders/service.go

```diff
@@ -707,6 +707,72 @@ func (service *Service) CreatePutRepairOrderLimits(ctx context.Context, bucketID
 	return limits, piecePrivateKey, nil
 }
 
+// CreateGracefulExitPutOrderLimit creates an order limit for graceful exit put transfers.
+func (service *Service) CreateGracefulExitPutOrderLimit(ctx context.Context, bucketID []byte, nodeID storj.NodeID, pieceNum int32, rootPieceID storj.PieceID, shareSize int32) (limit *pb.AddressedOrderLimit, _ storj.PiecePrivateKey, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	orderExpiration := time.Now().Add(service.orderExpiration)
+
+	piecePublicKey, piecePrivateKey, err := storj.NewPieceKey()
+	if err != nil {
+		return nil, storj.PiecePrivateKey{}, Error.Wrap(err)
+	}
+
+	serialNumber, err := service.createSerial(ctx)
+	if err != nil {
+		return nil, storj.PiecePrivateKey{}, Error.Wrap(err)
+	}
+
+	node, err := service.overlay.Get(ctx, nodeID)
+	if err != nil {
+		return nil, storj.PiecePrivateKey{}, Error.Wrap(err)
+	}
+
+	if node.Disqualified != nil {
+		return nil, storj.PiecePrivateKey{}, overlay.ErrNodeDisqualified.New("%v", nodeID)
+	}
+
+	if !service.overlay.IsOnline(node) {
+		return nil, storj.PiecePrivateKey{}, overlay.ErrNodeOffline.New("%v", nodeID)
+	}
+
+	orderLimit, err := signing.SignOrderLimit(ctx, service.satellite, &pb.OrderLimit{
+		SerialNumber:     serialNumber,
+		SatelliteId:      service.satellite.ID(),
+		SatelliteAddress: service.satelliteAddress,
+		UplinkPublicKey:  piecePublicKey,
+		StorageNodeId:    nodeID,
+		PieceId:          rootPieceID.Derive(nodeID, pieceNum),
+		Action:           pb.PieceAction_PUT_GRACEFUL_EXIT,
+		Limit:            int64(shareSize),
+		OrderCreation:    time.Now(),
```

Comment:
should this be time.Now().UTC() or okay to use time.Now()?

Reply:
Updated